pretty_midi/pretty_midi.py

```diff
@@ -306,20 +306,37 @@ def __get_instrument(program, channel, track, create_new):
                     # Store this as the last note-on location
                     note_on_index = (event.channel, event.note)
                     last_note_on[note_on_index].append((
-                        self.__tick_to_time[event.time],
-                        event.velocity))
+                        event.time, event.velocity))
```

Comment:
Any specific reason you are changing from time to ticks here?  I don't think floating point issues would bite us here unless they are nondeterministic :)